src/utils/addresses.ts

```
@@ -1,7 +1,11 @@
 import { isAddress } from 'viem';
-
+import { isDeployedAddress } from "@/utils/addresses";
 // A commonly used constant for the zero address
 export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+export const isDeployedAddress = (
```

Biome (2.4.4)

[error] Shouldn't redeclare 'isDeployedAddress'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)